packages/edit-navigation/src/hooks/use-navigation-editor-insertion-point.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+
+const useNavigationEditorInsertionPoint = () => {
+	return useSelect( ( select ) => {
+		const { getBlockOrder } = select( blockEditorStore );
+
+		return {
+			rootClientId: getBlockOrder()[ 0 ],
```

Comment:
I think this should only be the case when no block is selected or the nav block is selected, otherwise it should be possible to fall back to the block library default behavior of inserting after the selected block.